iree/compiler/Dialect/Flow/Transforms/test/dispatch_linalg_on_tensors.mlir

```diff
@@ -658,6 +658,55 @@ func @inline_dag_2(
 
 // -----
 
+func @inline_dag_3(%240 : tensor<9xi32>, %244 : tensor<18xi32>, %247 : tensor<i32>) -> tensor<9xi1> {
+  %c9 = constant 9 : index
+  %c5_i32 = constant 5 : i32
+  %c0_i32 = constant 0 : i32
+  %c9_i32 = constant 9 : i32
+  %245 = flow.tensor.update %240, %244[%c9] : tensor<9xi32> -> tensor<18xi32>
+  %248 = tensor.extract %247[] : tensor<i32>
+  %249 = cmpi slt, %248, %c9_i32 : i32
+  %250 = select %249, %248, %c9_i32 : i32
+  %251 = cmpi sgt, %250, %c0_i32 : i32
+  %252 = select %251, %250, %c0_i32 : i32
+  %253 = index_cast %252 : i32 to index
+  %254 = subtensor %245[%253] [9] [1] : tensor<18xi32> to tensor<9xi32>
+  %255 = linalg.init_tensor [9] : tensor<9xi1>
+  %256 = linalg.generic {
+      indexing_maps = [affine_map<(d0) -> (d0)>, affine_map<(d0) -> (d0)>],
+      iterator_types = ["parallel"]}
+      ins(%254 : tensor<9xi32>) outs(%255 : tensor<9xi1>) {
+        ^bb0(%arg20: i32, %arg21: i1):  // no predecessors
+          %849 = cmpi eq, %arg20, %c5_i32 : i32
+          linalg.yield %849 : i1
+      } -> tensor<9xi1>
+  return %256 : tensor<9xi1>
+}
```

Comment:
Could you pass this to `iree-opt` and use the output here? It will rename the variables with better naming. I think we don't want something like `%245` in the test.

Reply:
its just some number. I am not sure it matters :)

Reply:
I think it's just my preference. I'd prefer using a canonicalized form if we don't have meaningful names for them. :-)